app/views/bulk_actions/forms/_set_governing_apo_form.html.erb

```diff
@@ -1,12 +1,6 @@
-<div>
-  <%= f.fields_for :set_governing_apo do |set_governing_apo_fields| %>
-    <div class='form-group'>
-      <div class='text'>
```

Comment:
I see this text div was dropped in the changes to this file. Not needed?

Reply:
Nope, it wasn't doing anything useful.  It didn't have a style.

Reply:
@jcoyne ok, great!